checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -279,6 +283,12 @@ def evaluate_non_rendered_values(self) -> None:
             self.local_graph.update_vertex_config(vertex, changed_attributes)
 
     def evaluate_value(self, val: Any) -> Any:
+        val_length: int = len(str(val))
+        if CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```

Comment:
either
```python
if 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```
or 
```python
if CHECKOV_RENDER_MAX_LEN and CHECKOV_RENDER_MAX_LEN < val_length
```
and then `0` is the value to render any length

Reply:
First option is no good - CHECKOV_RENDER_MAX_LEN can be `None`. I'll use the second

Reply:
ah ok, then how about
```python
if CHECKOV_RENDER_MAX_LEN is not None and 0 < CHECKOV_RENDER_MAX_LEN < val_length:
```